lib/wanda_web/controllers/execution_controller.ex

```diff
@@ -59,4 +75,31 @@ defmodule WandaWeb.ExecutionController do
 
     render(conn, execution: execution)
   end
+
+  def start(
+        conn,
+        _params
+      ) do
+    %{
+      execution_id: execution_id,
+      group_id: group_id,
+      targets: targets,
+      env: env
+    } = Map.get(conn, :body_params)
+
+    case Server.start_execution(execution_id, group_id, Target.from_list(targets), env) do
```

Comment:
Let's use `execution_server_impl().start_execution(` to make the future testing better.
```
defp execution_server_impl,
    do: Application.fetch_env!(:wanda, Wanda.Policy)[:execution_server_impl]
```

Reply:
Changed 44a9eec360b38b02a997c84b527266137f459b95